src/pushsource/_impl/model/erratum.py

```diff
@@ -265,6 +266,35 @@ class ErratumPushItem(PushItem):
     :type: list[ErratumPackageCollection]
     """
 
+    container_list = attr.ib(
+        type=list, default=attr.Factory(frozenlist), converter=frozenlist
+    )
+    """A list of container images associated with the advisory.
+
+    Example of the list:
+    
+    .. code-block:: python
```

Comment:
You need a blank line after the code-block as well.  It should be possible to test this by running `tox -e docs` and checking the generated docs.

```suggestion
    .. code-block:: python

```